### PR TITLE
Pub 2184 green colour indicator to signify when password criteria is met

### PR DESCRIPTION
**Pull request by MindGames2015**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/PUB-2345

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
   </title>
   <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
   <meta name="theme-color" content="#0b0c0c">
-  <link rel="shortcut icon" sizes="16x16 32x32 48x48" href="{StorageAccountUrl}{B2cContainer}/favicon.ico"
+  <link rel="shortcut icon" sizes="16x16 32x32 48x48"
+        href="https://pipsssastg.blob.core.windows.net/b2c-custom-policy-files/favicon.ico"
```
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw-cy-GB.html:10]: These need to be changed back to the dynamic values previously, as they're replaced by infrastructure (there's a few below which are similar)

File: infrastructure/resources/b2c-polices/reset-pw-en.html
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw-en.html:1]: This looks like a copy of the HTML file when rendered by Azure.

This need to be changed back to the current file and only contain the changes needed for this new functionality.

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html [hunk 2]
```diff
@@ -65,20 +68,84 @@
...
 
 
   <main class="govuk-main-wrapper " id="main-content" role="main">
 
-    <link href="{StorageAccountUrl}{B2cContainer}/reset-pw.css" rel="stylesheet">
-    <div id="api">
-
+    <link href="https://pipsssastg.blob.core.windows.net/b2c-custom-policy-files/reset-pw.css" rel="stylesheet">
+    <div class="govuk-grid-row">
+      <div id="api" data-name="SelfAsserted" class="govuk-grid-column-two-thirds">
```
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw-cy-GB.html:81]: I do not believe this should be changed. the div for id="api" is injected into by Azure. The below code in this dev element I believe is from that injection, and therefore can be removed. I think it's only the div on line 139 that is the new block here?

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html (hunk 1 above)
Comment (MindGames2015) [reply infrastructure/resources/b2c-polices/reset-pw-cy-GB.html]: Changes made. Thanks Chris

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html (hunk 2 above)
Comment (MindGames2015) [reply infrastructure/resources/b2c-polices/reset-pw-cy-GB.html:71]: Changes made. Thanks Chris.

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html [hunk 3]
```diff
@@ -134,7 +191,7 @@ <h2 class="govuk-visually-hidden">Support links</h2>
           </li>
         </ul>
 
-        <img src="{StorageAccountUrl}{B2cContainer}/copyright.png" height="17" width="41"
+        <img src="{StorageAccountUrl}{B2cContainer}/logo_gov.png" height="17" width="41"
```
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw-cy-GB.html:194]: Not sure if this should have been changed - should this still be copyright?

File: infrastructure/resources/b2c-polices/reset-pw-cy-GB.html (hunk 2 above)
Comment (ChrisS1512) [reply infrastructure/resources/b2c-polices/reset-pw-cy-GB.html:71]: Un-resolving as issue still exists. I believe Everything within the div tag starting on line 71 until it's close on line 128 is auto injected by Azure so they do not need adding here, and only the div tag itself should exist. Were there any changes in this div block specific to the new functionality?

File: infrastructure/resources/b2c-polices/reset-pw-en.html [hunk 4]
```diff
@@ -70,15 +68,77 @@
...
   </nav>
 
-
   <main class="govuk-main-wrapper " id="main-content" role="main">
 
     <link href="{StorageAccountUrl}{B2cContainer}/reset-pw.css" rel="stylesheet">
-    <div id="api">
-
+    <div class="govuk-grid-row">
+      <div id="api" data-name="SelfAsserted" class="govuk-grid-column-two-thirds">
```
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw-en.html:75]: Same comment here as above regarding div tag

File: infrastructure/resources/b2c-polices/reset-pw.css [hunk 5]
```diff
@@ -3942,7 +3942,7 @@ div .attrEntry div + .govuk-heading-s,
   display: inline-block;
   min-width: 125px;
   padding-top: 112px;
-  background-image: url("{StorageAccountUrl}{B2cContainer}/govuk-crest.png");
+  background-image: url("https://pipsssastg.blob.core.windows.net/b2c-custom-policy-files/govuk-crest.png");
```
Comment (ChrisS1512) [inline infrastructure/resources/b2c-polices/reset-pw.css:3945]: This also needs changing back to the parameterised version. Same on line 3955

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**